orangecontrib/text/import_documents.py

```diff
@@ -156,19 +165,78 @@ def read_file(self):
         self.content = soup.get_text()
 
 
+class CsvMetaReader(Reader):
+    ext = [".csv"]
+
+    def read_file(self):
+        self.content = pd.read_csv(self.path)
+
+
+class YamlMetaReader(Reader):
+    ext = [".yaml"]
+
+    def read_file(self):
+        with open(self.path, "r") as f:
+            self.content = yaml.safe_load(f)
+
+
+class UrlReader(Reader, CoreUrlReader):
+    ext = [".url"]
+
+    def __init__(self, path, *args):
+        CoreUrlReader.__init__(self, path)
+        Reader.__init__(self, self.filename, *args)
+
+    def read_file(self):
+        path, name = os.path.split(self.filename)
+        self.filename = os.path.join(path, quote(name))
```

Comment:
This one is causing errors on windows. `os.path.join` joins URL together with backslashes on Windows. If I replace it with `self.filename = f"{path}/{quote(name)}" 
` it works fine.

Reply:
Thanks for the tip.